### PR TITLE
Create Rule “why-vue-is-great/rule”

### PR DESCRIPTION
**Pull request by zacharykeeping**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/why-vue-is-great/rule.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+uri: why-vue-is-great
+authors:
+  - title: Zach Keeping
+    url: https://ssw.com.au/people/zach-keeping/
+created: 2023-07-12T00:20:54.249Z
+guid: 0198bbf5-8c26-4ac2-b76d-7da4ccaa4917
+---
+Vue.js is an open-source JavaScript framework, similar to React and Angular. It was originally released by ex-Googler Evan You in 2014 who wanted to take what he liked about AngularJS to build a newer and more lightweight framework. Today, Vue.js is a fairly popular and mature framework with an ever-increasing market share behind React and Angular. Here are some of the reasons why Vue.js is great.
+
+**Easy to learn**
```
Comment (tiagov8) [inline rules/why-vue-is-great/rule.md:13]: 1. Please use headings instead of bolds

File: rules/why-vue-is-great/rule.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Vue.js is remarkably lightweight. A minimal Hello World application in Vue would be only around 16KB in size, whereas a similar Vue project with every optional feature enabled would sit at around just 27KB. This is much smaller in size compared to similar frameworks such as Angular and React.
+
+**High Performance**
+
+While good performance can be expected from a well-coded app using any of the major web frameworks, benchmarks have shown Vue.js edges out ahead of Angular and React in many metrics, such as memory usage, startup time, and general DOM manipulation. A developer should expect good performance from a Vue.js app out of the box.
+
+**Great Tooling**
+
+To set up a Vue.js project, it's as easy as using "create-tool" with Vite and following a few prompts to scaffold an application with all the features you need out of the box. Once set up, the official **Vue Devtools** extension for Chrome and Firefox allows for easy debugging and inspecting your Vue.js app.
```
Comment (tiagov8) [inline rules/why-vue-is-great/rule.md:27]: 1. Please add a link to "official Vue Dev tools"

File: rules/why-vue-is-great/rule.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+type: rule
+title: Do you know what makes Vue.js great?
+uri: why-vue-is-great
+authors:
+  - title: Zach Keeping
+    url: https://ssw.com.au/people/zach-keeping/
+created: 2023-07-12T00:20:54.249Z
+guid: 0198bbf5-8c26-4ac2-b76d-7da4ccaa4917
+---
+Vue.js is an open-source JavaScript framework, similar to React and Angular. It was originally released by ex-Googler Evan You in 2014 who wanted to take what he liked about AngularJS to build a newer and more lightweight framework. Today, Vue.js is a fairly popular and mature framework with an ever-increasing market share behind React and Angular. Here are some of the reasons why Vue.js is great.
```
Comment (tiagov8) [inline rules/why-vue-is-great/rule.md:11]: 1. Change fullstop to colon at the end of this sentence

File: rules/why-vue-is-great/rule.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+**Easy to learn**
+
+Compared to Angular and even React, Vue.js has a very short learning curve and makes it every easy to pick up regardless of skill level. Someone with fairly basic knowledge of JavaScript, HTML and CSS should be able to start up a Vue.js project with relative ease, making this a great framework for beginners or as an alternative for someone coming across from another framework. The official Vue.js website also provides thorough and easy-to-follow documentation, and a thriving online community of Vue.js developers can help one get a Vue.js web application off the ground in no time.
+
+**Lightweight**
+
+Vue.js is remarkably lightweight. A minimal Hello World application in Vue would be only around 16KB in size, whereas a similar Vue project with every optional feature enabled would sit at around just 27KB. This is much smaller in size compared to similar frameworks such as Angular and React.
```
Comment (tiagov8) [inline rules/why-vue-is-great/rule.md:19]: 1. Please put "Hello World" in quotes

File: rules/why-vue-is-great/rule.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+title: Do you know what makes Vue.js great?
+uri: why-vue-is-great
+authors:
+  - title: Zach Keeping
+    url: https://ssw.com.au/people/zach-keeping/
+created: 2023-07-12T00:20:54.249Z
+guid: 0198bbf5-8c26-4ac2-b76d-7da4ccaa4917
+---
+Vue.js is an open-source JavaScript framework, similar to React and Angular. It was originally released by ex-Googler Evan You in 2014 who wanted to take what he liked about AngularJS to build a newer and more lightweight framework. Today, Vue.js is a fairly popular and mature framework with an ever-increasing market share behind React and Angular. Here are some of the reasons why Vue.js is great.
+
```
[the remark this answers is not in the document]
Comment (zacharykeeping) [reply rules/why-vue-is-great/rule.md:12]: Done - And yep it's for the blurb. Must have went missing when I copied between editors. :)

**Comment by zacharykeeping** [comment]
> Hey @zacharykeeping
> 
> 1. Can you find a short (< 5min) video about Vue and add to the rule?
> 2. Rule is missing a category. Please add it to 1

Done all - all comments should be resolved now. :)

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**